ui/appui-abstract/eslint.config.js

```diff
@@ -0,0 +1,24 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
+  {
+    files: ["**/*.ts"],
+    ...iTwinPlugin.configs.iTwinjsRecommendedConfig,
+  },
+  {
+    files: ["**/*.ts"],
+    ...iTwinPlugin.configs.jsdocConfig,
```

Comment:
@Jake-Screen Should `uiConfig` also be imported for this package?

Reply:
Since this is not a test app i do not believe so